src/parquet/column/reader.h

```diff
@@ -103,17 +118,29 @@ class ColumnReader {
 
   parquet::PageHeader current_page_header_;
 
-  // Not set if field is required.
+  // Not set if field is required (flat schemas), or if the whole schema tree
```

Comment:
This is not set if the entire schema is required? The flat or not shouldn't matter right?